City-of-Helsinki/drupal-hdbt
# UHF-249 Basic page layout

To test:

1. `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main yoursite --no-interaction --repository https://repository.drupal.hel.ninja/`
2. Switch to corresponding branch: `composer require drupal/hdbt:dev-UHF-249-basic-page-layout`
3. Run `make new`
4. Go to any of the basic pages. Add some text-paragraphs, headers to the text, link paragraph. Check that everything looks good and works responsively. You can check the text-component functionality from [here](https://app.abstract.com/projects/6a2238d2-fcb7-40e4-b2aa-a4487941ad8a/branches/master/collections/cfe72cd0-ce9c-46fd-84b9-b917c6efe173)
5. Edit one of the basic pages and make sure the ckeditor uses helsinki-styles (helsinki grotesk font and same spacing, sizing etc)

Also check this PR for helfi-setup:
https://github.com/City-of-Helsinki/drupal-helfi/pull/36
